server/src/cloud/util.ts

```diff
@@ -1,32 +1,42 @@
 export interface ParseObjectParams {
-    [key: string]: any;
+  [key: string]: any;
 }
 
-export const api = <RequestData extends ParseObjectParams = ParseObjectParams, ResponseType = void>(name: string, call: (user: Parse.User, request: RequestData) => ResponseType | Promise<ResponseType>) => {
-    Parse.Cloud.define<(params: RequestData) => ResponseType | Promise<ResponseType>>(name, (request) => {
-        if (!request.user) {
-            throw new Error('Not authorized');
-        }
-        return call(request.user, request.params);
-    });
-}
+export const api = <RequestData extends ParseObjectParams = ParseObjectParams, ResponseType = void>(
+  name: string,
+  call: (user: Parse.User, request: RequestData) => ResponseType | Promise<ResponseType>
+) => {
+  Parse.Cloud.define<(params: RequestData) => ResponseType | Promise<ResponseType>>(name, (request) => {
+    if (!request.user) {
+      throw new Error("Not authorized");
+    }
+    return call(request.user, request.params);
+  });
+};
+
+export const authApi = <RequestData extends ParseObjectParams = ParseObjectParams, ResponseType = void>(
```

Comment:
`authApi` should be named `publicApi`, because it's not specifically related to the auth endpoints and can be called publiclly without api